static_src/actions/route_actions.js

```diff
@@ -108,5 +115,13 @@ export default {
       routeGuid,
       route
     });
+  },
+
+  error(routeGuid, err) {
```

Comment:
update, i read the rest, i can see the need for the two. just put the comment below in `constants.js`